### DISCUSSION TITLE
Cannot prove student status

### DISCUSSION THREAD

**Discussion by remadisson** [community/community#137108] [GitHub Education]
### Hi everyone,

I am trying to prove that I am a student (again) as my email from the university is no longer sufficient.

My university doesn't provide an ID with a valid date or a good proof that I could take a picture of with my mobile phone. So I'm trying to submit my transcript, but I can't because they only allow .jpg or .png and my transcript is 4 pages long, so how am I supposed to do that? - I've tried to stitch it together so it can still be read, but since it's a manipulation of a photo, it's no longer probative. 

_Oh and printing, isn't really an option, since you couldn't read it._

Do you have any idea how I can provide sufficient evidence?

**Comment by davevad93** [comment]
Hi @remadisson ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) "***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***"

So you can try one of the above, the most important thing is that your academic documents show a current date in a `DD/MM/YYYY` or `MM/DD/YYYY` format, otherwise your application could be rejected.

**Reply by remadisson** [reply]
Thank you, I've figured it out.

**Comment by zroussama** [comment]
same, that's sad
